apps/shelter/src/pages/volunteers/write/index.tsx

```diff
@@ -1,3 +1,80 @@
+import {
+  Box,
+  Button,
+  Center,
+  Flex,
+  FormControl,
+  FormHelperText,
+  FormLabel,
+  Input,
+  InputGroup,
+  InputRightAddon,
+  Textarea,
+} from '@chakra-ui/react';
+import { useState } from 'react';
+import EditPhotoList from 'shared/components/EditPhotoList';
+
+const DUMMY_IMAGE = 'https://source.unsplash.com/random';
+const DUMMY_IMAGE_URLS = Array.from({ length: 2 }, () => DUMMY_IMAGE);
+
 export default function VolunteersWritePage() {
-  return <h1>VolunteersWritePage</h1>;
+  const [imageUrls, setImageUrls] = useState<string[]>(DUMMY_IMAGE_URLS);
+
+  return (
+    <Box py={6} px={2}>
```

Comment:
```suggestion
    <Box pt={6} px={4}>
```

`px={4}`로 통일하기로 한 기억이 납니다! 🙋🏻‍♂️🤔